src/FakeItEasy/Creation/IProxyGenerator.cs

```diff
@@ -19,10 +19,10 @@ internal interface IProxyGenerator
         /// <param name="typeOfProxy">The type of proxy to generate.</param>
         /// <param name="additionalInterfacesToImplement">Interfaces to be implemented by the proxy.</param>
         /// <param name="argumentsForConstructor">Arguments to pass to the constructor of the type in <paramref name="typeOfProxy" />.</param>
-        /// <param name="customAttributeBuilders">The custom attribute builders.</param>
+        /// <param name="attributes">The expressions that describe the custom attributes to build into the proxy.</param>

```

Comment:
nitpick: "custom" seems redundant (it's not used in equivalent comments elsewhere).